android/BOINC/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
```

Comment:
This change doesn't relate to this PR either. Please move to a separate one. Thanks